internal/Network/DNS/Types/Internal.hs

```diff
@@ -1074,6 +1077,9 @@ instance Show RData where
       showTLSA usage selector mtype digest =
           show usage ++ " " ++ show selector ++ " " ++
           show mtype ++ " " ++ _b16encode digest
+      showCAA flags tag value =
+          show flags ++ " " ++ show tag ++ " " ++
```

Comment:
The `tag` is required to be alphanumeric ASCII, and its presentation form does not employ double-quotes.  Using `show` here, will emit the `tag` in double quotes.  Unless the tag does not satisfy the required syntax, it should probably be presented without quotes.